wpt-metadata/chromedriver/headless/webdriver/tests/bidi/browsing_context/set_viewport/set_viewport.py.ini

```diff
@@ -0,0 +1,3 @@
+[set_viewport.py]
+  [test_set_viewport_reset]
+    disabled: flaky
```

Comment:
To mark as flaky: `expected: [PASS, ERROR]` is more explicit. Or `FAIL` instead of `ERROR` depending on the circumstance.

Reply:
Do we really want to run test if we don't care about it's result? I find disabling with a mark "flaky" more informative, as it does not give a false confidence after the test passed. Plus, disabling flaky tests reduces the test runtime. 

Of cause I'm open for discussion.

Reply:
I'm open to either approach.